boards/arm/atsame54_xpro/pinmux.c

```diff
@@ -106,6 +106,20 @@ static int board_pinmux_init(struct device *dev)
 	pinmux_pin_set(muxa, 25, PINMUX_FUNC_H);
 	pinmux_pin_set(muxa, 24, PINMUX_FUNC_H);
 #endif
+
+#if DT_ATMEL_SAM_GMAC_42000800_BASE_ADDRESS
```

Comment:
Use a node label

Reply:
@galak Sure, I will do that.

Do you want me to convert the rest of such in this file as well? or are you already doing it somewhere else?

Reply:
I have converted this change to use the new DT macros, and did not touch the rest here assuming that will be done in a separate mass conversion PR.